features/upgrade/routing/upgrade.feature

```diff
@@ -208,3 +208,82 @@ Feature: Routing and DNS related scenarios
     Then the step should succeed
     And the output should contain "Hello-OpenShift web-server-rc"
     """
+
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  Scenario: Unidling a route work without user intervention - prepare
```

Comment:
Perhaps add yourself as author ?

Reply:
i can see, auther name is not added in other testcases 'prepare' script, but i added now